userguide/virtualmachines.rst

```diff
@@ -183,7 +183,7 @@ a Virtual Machine (VM) type are described in
    |          |                    |                | number of CPUs.                                                                               |
    +----------+--------------------+----------------+-----------------------------------------------------------------------------------------------+
    | 2        | Memory Size        | integer        | Set the amount of RAM for the VM. Allocating too much memory can slow the system or           |
-   |          |                    |                | prevent VMs from running.                                                                     |
+   |          |                    |                | prevent VMs from running. This field is :ref:`human readable <Human Readable Fields>`.        |
```

Comment:
As above, with text replacement if possible (This is a human readable field) and remove the sentences with examples.